src/main/java/com/hladchenko/springrestapi/controller/NoteController.java

```diff
@@ -0,0 +1,30 @@
+package com.hladchenko.springrestapi.controller;
+
+import com.hladchenko.springrestapi.entity.Note;
+import com.hladchenko.springrestapi.entity.User;
+import com.hladchenko.springrestapi.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequestMapping("/notes")
+@RestController
+public class NoteController {
+
+    private final UserService userService;
+
+    public NoteController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/{uuid}")
+    public User addNote(@PathVariable UUID uuid, @RequestBody Note note) throws Exception {
+        return userService.addNote(uuid, note);
+    }
+
+    @GetMapping("/{uuid}")
```

Comment:
For me, it's a little bit ambiguous to you here user ID, if i would see result link i would think that it's request to get single note by specified note ID.
Compare same with `UserController` GET request.